File: TransCoder_tokenized_test_set_functions/java/PAIRS_OF_POSITIVE_NEGATIVE_VALUES_IN_AN_ARRAY.java
public static void printPairs ( int arr [ ], int n ) {
  Vector < Integer > v = new Vector < Integer > ( );
  for ( int i = 0;
  i < n;
  i ++ ) for ( int j = i + 1;
  j < n;
  j ++ ) if ( Math . abs ( arr [ i ] ) == Math . abs ( arr [ j ] ) ) v . add ( Math . abs ( arr [ i ] ) );
  if ( v . size ( ) == 0 ) return;
  Collections . sort ( v );
  for ( int i = 0;
  i < v . size ( );
  i ++ ) System . out . print ( - v . get ( i ) + " " + v . get ( i ) );
}

File: TransCoder_tokenized_test_set_functions/python/PAIRS_OF_POSITIVE_NEGATIVE_VALUES_IN_AN_ARRAY.py
def printPairs ( arr, n ) :
    v = [ ]
    for i in range ( n ) :
        for j in range ( i + 1, n ) :
            if ( abs ( arr [ i ] ) == abs ( arr [ j ] ) ) :
                v . append ( abs ( arr [ i ] ) )
    if ( len ( v ) == 0 ) :
        return;
        
    v . sort ( )
    for i in range ( len ( v ) ) :
        print ( - v [ i ], "", v [ i ], end = " " )
    
